CHANGE-NOTES.html

```diff
@@ -1,4 +1 @@
-<p>Fix docs on the shortcut that opens Git Machete Tab <i>(command option shift M)</i> on Mac.</p>
-
-<p><i>Merge Parent Into Current</i> action has been added.
-    The action merges a parent branch into the current branch, as suggested by @mkondratek and @pedroadame.</p>
+<p>Dropped support for IntelliJ 2020.2.</p>
```

Comment:
The other entries are also to be retained! As we'll probably just release v2.0.0, skipping v1.4.0 entirely